fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailFragment.java

```diff
@@ -153,17 +142,21 @@ private void setupEventListAdapter() {
 
   private void setupEventListUpdates() {
     viewModel
-        .getLaoEvents()
+        .getRollCalls()
+        .observe(
+            requireActivity(),
+            rollCalls -> {
+              Log.d(TAG, "Got a list update for roll call events " + rollCalls.toString());
+              mEventListViewEventAdapter.replaceRollCalls(rollCalls);
+            });
+
+    viewModel
+        .getElections()
         .observe(
             requireActivity(),
-            events -> {
-              Log.d(TAG, "Got an event list update");
-              for (Event event : events) {
-                if (event.getType() == EventType.ROLL_CALL) {
-                  Log.d(TAG, ((RollCall) event).getDescription());
-                }
-              }
-              mEventListViewEventAdapter.replaceList(events);
+            elections -> {
+              Log.d(TAG, "Got a list update for election events " + elections.toString());
+              mEventListViewEventAdapter.replaceElections(elections);
```

Comment:
Yes. Same as the comment above, once the election repo exists it should be elegantly solvable. With rx java one can combine observables if I'm not mistaken so that should be cool !
In fact the real question in my eyes is : should start to move away from MutableLiveData for Repository stuff. Cause I realised what we do do not make much sense : we observe an observable from the repo -> when it's triggered we update a `MutableLiveData` -> it triggers the `MutableLiveData`

Reply:
Yes, we probably need to use the live data only for specific stuff related to the view itself.